neo/sys/posix/platform_linux.cpp

```diff
@@ -56,21 +56,32 @@ Sys_EXEPath
 */
 const char* Sys_EXEPath()
 {
-	static char	buf[ 1024 ];
```

Comment:
it *is* hard to "fix", because the game uses strings with static length (of `MAX_OSPATH` chars) everywhere. replacing that with something dynamic is not that easy (and a waste of time, because I doubt anyone really uses paths with > 4096 chars in their filesystems. In a 80char terminal, 4096 chars are 51 lines. Maybe there are usecases for such long paths, but I don't think running a video game is one.